aea/configurations/base.py

```diff
@@ -1703,7 +1705,14 @@ def update(self, data: Dict) -> None:
         self.component_configurations = result
 
         # update other fields
-        # currently not supported.
+        for item_id, value in data.get("private_key_paths", {}).items():
+            self.private_key_paths.update(item_id, value)
```

Comment:
is it correct we only update existing keys?

Reply:
No, at the moment we need to be able to add and override, otherwise it is not very useful

Reply:
OK, the behaviour of `CRUDCollection.update` now creates an item if it the key doesn't exists yet.